### PR TITLE
chore: update CONTRIBUTING.md for new process

### PR DESCRIPTION
**Pull request by srl295**
- matches standard Unicode text

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -204,8 +204,17 @@ The following PR has one non-blocking review, one blocking review, one approval,
...
+
+- **Individual CLA**: If you have determined that the Individual CLA is appropriate, just open a Pull Request and you will have the opportunity to click to accept the Individual CLA.
+
+- **Corporate CLA**: If you have determined that a Corporate CLA is appropriate, please check the [public list of Corporate CLAs][unicode-corporate-clas] that the Consortium has on file. If your employer has already signed a CLA, then just open a Pull Request and you will have the opportunity to click that your employer has already signed a CLA. If your employer has not already signed a CLA, you will need to arrange for your employer to sign the Corporate CLA, as described in [How to Sign a Unicode CLA][signing].
+
+Unless otherwise noted in the LICENSE file, this project is released under the free and open-source [Unicode License][unicode-license], also known as Unicode, Inc. License Agreement - Data Files and Software.
```
Comment (Manishearth) [inline CONTRIBUTING.md:215]: can this line be in the above section so people who just want to know the license can see it?

**Review by Manishearth — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (srl295) [reply CONTRIBUTING.md:215]: @annebright ^ ?

Comment (skywalkerswartz) [reply CONTRIBUTING.md:215]: @Manishearth can you help me understand what you're asking for and why? This section is designed to be the same across all repositories, based on the master copy in https://github.com/unicode-org/.github/blob/main/.github/CONTRIBUTING.md so we'd need a really good reason to deviate (unless you're proposing we also change the master copy in some way?)

Comment (srl295) [reply CONTRIBUTING.md:215]: I wondered about adding another heading above this one with such a link. Before the CLa link that's standard.

Comment (Manishearth) [reply CONTRIBUTING.md:215]: @skywalkerswartz Actually, I don't think it's that necessary: I thought this was updating README, not CONTRIBUTING.

Basically, my contention was that a README file's "licenses" section should start with the actual license information (not considerations on inbound code).

But for CONTRIBUTING, centering the CLA is quite reasonable.

Comment (srl295) [reply CONTRIBUTING.md:215]: there's a separate activity for going over the README file's references. yes, that's the difference

**Pull request merged**